sw/device/tests/sim_dv/uart_tx_rx_test.c

```diff
@@ -491,6 +504,36 @@ static bool execute_test(const dif_uart_t *uart) {
   return true;
 }
 
+void config_external_clock(void) {
+  uint32_t extclk_ctrl_reg = 0;
+  dif_lc_ctrl_t lc;
+  dif_clkmgr_t clkmgr;
+  mmio_region_t lc_reg = mmio_region_from_addr(TOP_EARLGREY_LC_CTRL_BASE_ADDR);
+  mmio_region_t clkmgr_reg =
```

Comment:
Nit: I think past convention has been to call these `*_base_addr`, so `lc_ctrl_base_addr` and `clkmgr_base_addr` respectively.

Reply:
Updated, thanks